server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -733,38 +733,38 @@ func (e *EventChannel) isFirstStartedEvent(bazelBuildEvent *build_event_stream.B
 	if e.hasReceivedStartedEvent {
 		return false
 	}
-	_, ok := bazelBuildEvent.Payload.(*build_event_stream.BuildEvent_Started)
+	_, ok := bazelBuildEvent.GetPayload().(*build_event_stream.BuildEvent_Started)
```

Comment:
is this a has check?

Reply:
it's a type-cast check. Here we are checking if the `oneOf` payload is of `message Started` type.